code/modules/client/preference/preferences_mysql.dm

```diff
@@ -206,50 +207,51 @@
 		real_name = query.item[2]
 		be_random_name = text2num(query.item[3])
 		gender = query.item[4]
-		age = text2num(query.item[5])
-		species = query.item[6]
-		language = query.item[7]
-
-		h_colour = query.item[8]
-		h_sec_colour = query.item[9]
-		f_colour = query.item[10]
-		f_sec_colour = query.item[11]
-		s_tone = text2num(query.item[12])
-		s_colour = query.item[13]
-		m_colours = params2list(query.item[14])
-		hacc_colour = query.item[15]
-		h_style = query.item[16]
-		f_style = query.item[17]
-		m_styles = params2list(query.item[18])
-		ha_style = query.item[19]
-		alt_head = query.item[20]
-		e_colour = query.item[21]
-		underwear = query.item[22]
-		undershirt = query.item[23]
-		backbag = query.item[24]
-		b_type = query.item[25]
+		body_type = query.item[5]
+		age = text2num(query.item[6])
+		species = query.item[7]
+		language = query.item[8]
+
+		h_colour = query.item[9]
+		h_sec_colour = query.item[10]
+		f_colour = query.item[11]
+		f_sec_colour = query.item[12]
+		s_tone = text2num(query.item[13])
+		s_colour = query.item[14]
+		m_colours = params2list(query.item[15])
+		hacc_colour = query.item[16]
+		h_style = query.item[17]
+		f_style = query.item[18]
+		m_styles = params2list(query.item[19])
+		ha_style = query.item[20]
+		alt_head = query.item[21]
+		e_colour = query.item[22]
+		underwear = query.item[23]
+		undershirt = query.item[24]
+		backbag = query.item[25]
+		b_type = query.item[26]
```

Comment:
Absolutely not.

Put the column for this at the end of the select statement to avoid creating conflict and problem hell here